### ISSUE TITLE
RF Bridge spontaneously made factory reset

### ISSUE THREAD

**Issue by Geroldswil** [xoseperez/espurna#1134]
Yesterday, my Sonoff RF Bridge (with 1.13.0) spontaneously reset itself to factory defaults by no visible reason. Any clues?

**Comment by xoseperez** [comment]
No, can you copy here the output for the `info` command in the web debug?

**Comment by Geroldswil** [comment]
Maybe not to useful as I upgraded to 1.13.1 in between...

info:

[432137] [WEBSOCKET] Requested action: dbgcmd
[432146]

[432148] [INIT] ESPURNA 1.13.1
[432150] [INIT] xose.perez@gmail.com
[432151] [INIT] http://tinkerman.cat

[432152] [INIT] CPU chip ID: 0xC6B03B
[432154] [INIT] CPU frequency: 80 MHz
[432155] [INIT] SDK version: 1.5.3(aec24ac9)
[432156] [INIT] Core version: 2.3.0
[432157] [INIT] Core revision:
[432159]
[432160] [INIT] Flash chip ID: 0x144051
[432163] [INIT] Flash speed: 40000000 Hz
[432164] [INIT] Flash mode: DOUT
[432165]
[432167] [INIT] Flash size (CHIP)   :  1048576 bytes /  256 sectors (   0
to  255)
[432169] [INIT] Flash size (SDK)    :  1048576 bytes /  256 sectors (   0
to  255)
[432171] [INIT] Reserved            :     4096 bytes /    1 sectors (   0
to    0)
[432172] [INIT] Firmware size       :   467696 bytes /  115 sectors (   1
to  115)
[432174] [INIT] Max OTA size        :   552960 bytes /  135 sectors ( 116
to  250)
[432175] [INIT] EEPROM size         :     4096 bytes /    1 sectors ( 251
to  251)
[432177] [INIT] Reserved            :    16384 bytes /    4 sectors ( 252
to  255)
[432178]
[432180] [INIT] EEPROM sectors: 251, 250
[432181]
[432185] [INIT] BOARD: ITEAD_SONOFF_RFBRIDGE
[432188] [INIT] SUPPORT: ALEXA BROKER DEBUG_TELNET DEBUG_WEB DOMOTICZ
HOMEASSISTANT MDNS_SERVER MQTT NTP SCHEDULER TELNET TERMINAL THINGSPEAK WEB
[432189] [INIT] WEBUI IMAGE CODE: 4
[432191]
[432192] [INIT] Last reset reason: Reboot from web interface
[432194] [INIT] Settings size: 2632 bytes
[432196] [INIT] Free heap: 10096 bytes






Ronny Colotto
Dorfstr. 74
8954 Geroldswil

079 215 21 90
ronny.colotto@gmail.com

**Comment by xoseperez** [comment]
How many switches do you have? The heap value is really low...
I have just committed a change to the `dev` branch that improved the memory usage when using several switches like the RF Bridge does.

**Comment by Geroldswil** [comment]
Currently, there are four switches configured and six events scheduled.

**Comment by xoseperez** [comment]
Scheduled events do not take memory... it is really weird you have so very little free heap.

**Comment by jsponz** [comment]
@xoseperez what should be a "good"/expected value for free heap?

**Comment by vaidyasr** [comment]
My RF Bridge too did auto factory reset yesterday.
```
[265616] [WEBSOCKET] #3 connected, ip: 192.168.4.2, url: /ws
[271836] [WEBSOCKET] Requested action: dbgcmd
[271845] 

[271846] [INIT] ESPURNA 1.13.1
[271847] [INIT] xose.perez@gmail.com
[271848] [INIT] http://tinkerman.cat

[271849] [INIT] CPU chip ID: 0x8E74E2
[271850] [INIT] CPU frequency: 80 MHz
[271851] [INIT] SDK version: 1.5.3(aec24ac9)
[271852] [INIT] Core version: 2.3.0
[271853] [INIT] Core revision: 
[271854] 
[300433] [MAIN] Uptime: 300 seconds
[300435] [MAIN] Free heap: 13168 bytes
[300437] [MAIN] Power: 3196 mV
```

**Comment by kakaki** [comment]
I have similar problem with sonoff basic - every time i try to save setting, like change led mode to "find me", it crashes and restores to factory settings (returns to default password), tried on 1.13.1 with fixed ip setting and latest version 1.13.2 without any modification - looks like some problems while saving eprom.
A have same problem with blitzwolf shp2 - can't modify any settings, it's reboots and restores to factory setting - after update with fixed ip one is working now for 4 days without any issue, and second one crashed and restored today in middle night.

**Comment by xoseperez** [comment]
Can you please get the info output and copy it here? Just go to the web UI, to the DEBUG tab and type `info` in the field, then send. Copy the output here, please.
